src/metaprob/syntax.clj

```diff
@@ -304,16 +304,16 @@
 (defn from-clojure-definition [exp]
   (let [[_ pattern rhs] exp
         key (if (and (symbol? pattern)
-                     (not (= pattern '_))
-                     (not (= pattern 'pattern)))
+                     (not= pattern '_)
+                     (not= pattern 'pattern))
```

Comment:
Not a style issue, but some day this should be reviewed for proper interaction with clojure namespaces.  There are a bunch of things like this, e.g. `&` and `define`.